src/__tests__/lib/command/util/apps-util.test.ts

```diff
@@ -183,10 +189,8 @@ describe('buildTableOutput', () => {
 		expect(newOutputTableMock).toHaveBeenCalledWith(
 			expect.objectContaining({ head: ['Key', 'Value'] }),
 		)
-		expect(pushMock).toHaveBeenCalledTimes(1)
-		expect(pushMock).toHaveBeenCalledWith(['setting', 'setting value'])
-		expect(toStringMock).toHaveBeenCalledTimes(1)
-		expect(toStringMock).toHaveBeenCalledWith()
+		expect(pushMock).toHaveBeenCalledExactlyOnceWith(['setting', 'setting value'])
+		expect(toStringMock).toHaveBeenCalledExactlyOnceWith()
```

Comment:
I checked the Jest documentation but I'm still not quite sure about something.  Is this in fact checking that `toStringMock` was called exactly once with no parameters?  Or is it not checking parameters since none were supplied?

Reply:
A call to a `With` function that doesn't check the `With` seems rather silly so checking for no arguments is the only thing that makes sense to me. It's easy enough to test though; I just removed the argument from a call that had one and I get an error. Here's the output when I remove the argument from line 192 above:

```
 FAIL  src/__tests__/lib/command/util/apps-util.test.ts
  ● buildTableOutput › creates new table with correct options and adds settings

    expect(received).toHaveBeenCalledExactlyOnceWith(expected)

    Expected mock function to have been called exactly once with [], but it was called with ["setting", "setting value"]

      190 | 			expect.objectContaining({ head: ['Key', 'Value'] }),
      191 | 		)
    > 192 | 		expect(pushMock).toHaveBeenCalledExactlyOnceWith()
          | 		                 ^
      193 | 		expect(toStringMock).toHaveBeenCalledExactlyOnceWith()
      194 | 	})
      195 | })

      at Object.<anonymous> (src/__tests__/lib/command/util/apps-util.test.ts:192:20)

Test Suites: 1 failed, 2 passed, 3 total
Tests:       1 failed, 58 passed, 59 total
Snapshots:   0 total
Time:        2.992 s, estimated 6 s
Ran all test suites related to changed files.
```